frontend/pages/software/ManageSoftwarePage/ManageSoftwarePage.tsx

```diff
@@ -663,7 +658,7 @@ const ManageSoftwarePage = ({
         renderCount={renderSoftwareCount}
         renderFooter={renderTableFooter}
         disableMultiRowSelect
-        onSelectSingleRow={handleRowSelect}
+        {...{ resetPageIndex, searchable, onQueryChange, onSelectSingleRow }}
```

Comment:
I thought we were sticking with the existing, more verbose pattern unless all props are being spread unchanged?

Reply:
I thought we were alright with this hybrid style...let's discuss at frontend sync